app/src/main/java/org/mozilla/fenix/tabstray/TrayPagerAdapter.kt

```diff
@@ -42,7 +40,7 @@ class TrayPagerAdapter(
             InactiveTabsAdapter(
                 lifecycleOwner = lifecycleOwner,
                 tabsTrayStore = tabsTrayStore,
-                inactiveTabsInteractor = inactiveTabsInteractor,
+                inactiveTabsInteractor = tabsTrayInteractor,
```

Comment:
IMO, it would make sense to update the parameters names to match this change as well. Would this be an appropriate time to do so for this object hierarchy? Looking through the meta issues, it wasn't immediately clear to me whether this would be revisited.